src/librustc/mir/mod.rs

```diff
@@ -53,7 +53,7 @@ pub mod traversal;
 pub mod visit;
 
 /// Types for locals
-type LocalDecls<'tcx> = IndexVec<Local, LocalDecl<'tcx>>;
+type LocalDecls<'tcx> = IndexVec<LocalWithRegion, LocalDecl<'tcx>>;
```

Comment:
This should remain `Local`! This is the full set of local declarations -- `LocalWithRegion` is meant to be a subset of the full set of locals.